stdlib/Statistics/test/runtests.jl

```diff
@@ -86,6 +86,19 @@ end
     @test ismissing(mean([missing, NaN]))
     @test isequal(mean([missing 1.0; 2.0 3.0], dims=1), [missing 2.0])
     @test mean(skipmissing([1, missing, 2])) === 1.5
+    @test isequal(mean(Complex{Float64}[]), NaN+NaN*im)
+    @test isequal(mean(skipmissing(Complex{Float64}[])), NaN+NaN*im)
+    @test mean(Complex{Float64}[]) isa Complex{Float64}
```

Comment:
Why not also test the value? Same in other places.

Reply:
The value is tested above in line 89 (unless you meant something else).

Reply:
Sorry, this one is, I originally spotted `mean(Int[])`. It would probably be clearer to have them side by side.